internal/pkg/installer/installer.go

```diff
@@ -216,37 +188,3 @@ func wipe(manifest *manifest.Manifest) (err error) {
 
 	return nil
 }
-
-func exists(devpath string) (bool, error) {
```

Comment:
IMO this should not be removed. Determining if an installation is present based solely on labels isn't reliable. I also had plans in the back on my mind to expand on the file that gets dropped into the boot partition to include some metadata about the install.

Reply:
> Determining if an installation is present based solely on labels isn't reliable

Why's that? The only instance I can think of is if the installation itself failed. But now that we minimally use the disk, I'm not sure there's much risk here since it's mostly ephemeral.